### PR TITLE
don't switch BS for prompt

### PR DESCRIPTION
**Pull request by deguio**

### CODE REVIEW

**Comment by deguio** [comment]
please test

File: Configuration/DataProcessing/python/RecoTLR.py [hunk 1]
```diff
@@ -186,7 +186,6 @@ def customiseExpressHI(process):
 ##############################################################################
 def customisePromptHI(process):
     process = customiseCommonHI(process)
-    process = _swapOfflineBSwithOnline(process)
```
Comment (slava77) [inline Configuration/DataProcessing/python/RecoTLR.py]: why is this needed?
This customisePromptHI is kept for Run1 tests and replays.

**Comment by slava77** [comment]
@deguio I suggest to close this one.
The modifications are done to the run1 setup and I'd rather preserve it as is, unless there are good reasons to change (like we will be rerunning reco of 2011 data)

**Comment by deguio** [comment]
makes sense to close then.
cheers,
F.
